src/main/java/de/retest/ui/descriptors/IdentifyingAttributes.java

```diff
@@ -59,9 +58,16 @@ public IdentifyingAttributes( final Collection<Attribute> attributes ) {
 		}
 	}
 
-	public static Collection<Attribute> createList( final Path path, final String type ) {
+	public static Collection<Attribute> createList( final Path path, String type ) {
+		if ( type == null ) {
+			throw new NullPointerException( "Type must not be null." );
+		}
+		type = type.trim();
+		if ( type.isEmpty() ) {
+			throw new IllegalArgumentException( "Type may not be empty." );
```

Comment:
Above: "… must not …", here: "… may not …". Would use the same way ("may not" or "must not").